crates/ty_project/src/metadata/options.rs

```diff
@@ -271,15 +334,31 @@ pub struct EnvironmentOptions {
     ///
     /// This option is commonly used to specify the path to a virtual environment.
```

Comment:
As of today, it _has_ to be a virtual environment, or else we error because we can't find `pyvenv.cfg`. This is a limitation we should fix, so maybe the text here can reflect the way it should work, not the current limitations.

Reply:
Makes sense. I don't think I'm the right person to write this because my understanding of the entire venv/system python is very limited. @AlexWaygood could you help me out here and pep up the description (in a separate PR?)

Reply:
Absolutely

Reply:
Thank you!